.github/workflows/deploy-stubs.yml

```diff
@@ -44,13 +44,16 @@ jobs:
         with:
           path: .aws-sam/cache
           key: orch-sam
+      
+      - name: Install dependencies
+        run: npm install
```

Comment:
I think `npm ci` is more appropriate here than `npm install`, see https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci

Reply:
Will have to look further on this - maybe could be used in another place as it always deletes node_modules. Unsure interaction with caching.